packages/block-editor/src/components/block-list/block.js

```diff
@@ -240,6 +237,14 @@ function BlockListBlock( {
 		placeCaretAtHorizontalEdge( target, isReverse );
 	};
 
+	useEffect( () => {
+		wrapper.current.addEventListener( 'dragstart', preventDrag );
+
+		return () => {
+			wrapper.current.removeEventListener( 'dragstart', preventDrag );
+		};
+	} );
```

Comment:
I think you might be able to optimize this?

```suggestion
	}, [ wrapper.current ] );
```

Reply:
Thanks, very good.